numba/pycc/cc.py

```diff
@@ -246,7 +246,7 @@ def distutils_extension(self, **kwargs):
         library_dirs = (kwargs.pop('library_dirs', [])
                         + self._toolchain.get_python_library_dirs())
 
-        ext = _CCExtension(name=self._basename,
+        ext = _CCExtension(name=self._source_module[:self._source_module.rfind('.')+1] + self._basename,
```

Comment:
I understand that it does look very neat to have all of this on one line. I am just slightly concerned about readability, so I would advocate using an additional variable here to hold the `self._source_module[:self._source_module.rfind('.')+1]` part.

Reply:
done